bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -109,6 +115,7 @@ <h2 id="payouts-section">Payouts</h2>
     <li>For *.mozilla.org, *.mozilla.com, *.mozilla.net, and *.firefox.com.</li>
     <li>Lack of clickjacking protection (XFO, CSP) is insufficient to claim a bounty</li>
     <li>Requires a proof of concept to be eligible</li>
+    <li>SSRF must be proven to reach internal network hosts, pinging Burp Collaborator and performing DNS lookups are not enough to prove SSRF</li>
```

Comment:
Maybe emphasize **internal**.

We could also add a sentence *why* we don't consider pings and dns lookups enough (I assume there are middle-boxes and reverse proxies that sometimes do unavoidable requests which we don't care about? Maybe we could link to an e.g., akamai support page from here?)

Reply:
hmm not sure where to find a reference page for this